soc/nordic/nrf54h/Kconfig

```diff
@@ -71,3 +73,23 @@ config SOC_NRF54H20_NO_MRAM_LATENCY
 	imply NRFS
 	imply NRFS_MRAM_SERVICE_ENABLED
 	default y if SOC_NRF54H20_CPUAPP
+
+config SOC_NRF54H20_GLOBAL_HSFLL_RESTRICT_MIN_FREQ
+	bool "Restrict minimum global HSFLL clock frequency"
+	select NRFS
+	select NRFS_GDFS_SERVICE_ENABLED
+	select CLOCK_CONTROL
+	default y if SOC_NRF54H20_CPUAPP
+
+if SOC_NRF54H20_GLOBAL_HSFLL_RESTRICT_MIN_FREQ
+
+config SOC_NRF54H20_GLOBAL_HSFLL_MIN_FREQ_HZ
+	int "Minimum global HSFLL clock frequency in Hertz"
+	range 64000000 320000000
+	default 320000000
```

Comment:
Having a range here is pretty pointless IMHO because the only allowed frequencies are: 64, 128, 256 and 320 MHz. Please change it into a choice of one of the allowed values.

Reply:
will do

Reply:
done

Reply:
How to make `SOC_NRF54H20_GLOBAL_HSFLL_MIN_FREQ_64_MHZ` selected with prj.conf? No matter what I try it always defaults to 320 MHz.

Reply:
`CONFIG_SOC_NRF54H20_GLOBAL_HSFLL_MIN_FREQ_64_MHZ=y`

Reply:
Thanks, turns out I was modifying wrong `prj.conf` file (I should have modified `usbd_next_prj.conf`).